app/crud/versions.py

```diff
@@ -155,6 +171,12 @@ async def _update_is_downloadable(
 
 
 async def _reset_is_latest(dataset: str, version: str) -> None:
+    """Set is_latest to False for all other versions of a dataset."""
+    # TODO: Should we make sure the version is valid to avoid setting nothing
+    # to latest? Or is being able to do that a desired feature?
```

Comment:
I think you can remove this TODO, and just add a line to the header comment that says it is recommended to set is_latest flag for version before calling this method to reset is_latest for all other versions [since that is what is done in update_version].